src/libs/logging/src/lib.rs

```diff
@@ -51,7 +52,8 @@ const DEFAULT_SUBSYSTEM: &str = "root";
 
 // Creates a logger which prints output as human readable text to the terminal
 pub fn create_term_logger(level: slog::Level) -> (slog::Logger, slog_async::AsyncGuard) {
-    let term_drain = slog_term::term_compact().fuse();
+    let decorator = TermDecorator::new().stdout().build();
+    let term_drain = slog_term::CompactFormat::new(decorator).build().fuse();
```

Comment:
Thx @kloud-byun Could you please do it as `TermDevorator` does?
```
-  let term_drain = slog_term::CompactFormat::new(decorator).build().fuse();
+ let term_drain = CompactFormat::new(decorator).build().fuse();

Import  and append `CompactFormat` in L16
+ use slog_term::{CompactFormat, TermDecorator};
```

Reply:
Fixed it, hopefully that looks ok now! :) Thanks for that.

Reply:
@Apokleos ah ha, I had no idea. Thank you so much for that. Learning a new thing again... appreciate it!